compliant-sets/many-operations/output.yaml

```
@@ -0,0 +1,109 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Swagger Petstore
+  license:
+    name: MIT
+servers:
+  - url: 'http://petstore.swagger.io/v1'
+paths:
+  /pets:
+    get:
+      operationId: listPets
+      tags:
+        - pets
+        - wildcard-done
+        - overlayed
+      parameters:
+        - name: limit
+          in: query
+          description: How many items to return at one time (max 100)
+          required: false
+          schema:
+            type: integer
+            maximum: 100
+            format: int32
+      responses:
+        '200':
+          description: A paged array of pets
+          headers:
+            x-next:
+              description: A link to the next page of responses
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Pets'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-info:
+        x-overlay-applied: structured-overlay
+        description: This is an added description
+    post: &ref_0
```

```suggestion
    post:
```
I would not expect/require an Overlay processor to do this kind of YAML magic, I'd rather expect that it simply merges this everywhere, especially since the `post` operation may already exist and the Overlay would just add `foo` next to existing properties.

---

I've removed this example from the pull request for now.